### ISSUE TITLE
Makisu cant pull `FROM` local docker cache?

### ISSUE THREAD

**Issue by bajacondor** [uber/makisu#281]
**Describe the bug**
When I specify an image that was added to the local docker registry in the a Dockerfile's `FROM` line, Makisu looks to `index.docker.io` before pulling from the local cache

**To Reproduce**
run the `build_makisu` function as described in the README with a dockerfile that specifies a local image in the `FROM` line

**Expected behavior**
Makisu should pull the image from the local cached registry

Thank you

**Comment by yiranwang52** [comment]
You mean an image that was pulled into local docker daemon?
That's not supported.

**Comment by bajacondor** [comment]
Yes.  Seems like if you were building multiple images you'd want to be able to cache base layers locally.  If the tool has a `--load` flag to push into the docker daemon, why not be able to pull from it?

**Comment by yiranwang52** [comment]
Makisu caches images locally in its directory, so there is no need.
Most interaction with docker daemon are also very slow.

**Comment by bajacondor** [comment]
So, then how do I load an image in the `FROM` line that was cached locally by Makisu?
